doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -719,17 +720,19 @@ the toplevel, and using them in source files is discouraged.
 
 .. cmd:: Add ML Path @string
 
-   This command adds the path :n:`@string` to the current |OCaml|
-   loadpath (cf. :cmd:`Declare ML Module`).
-
+   This command is equivalent to the command line option :n:`-I
+   @string` and adds the path :n:`@string` to the current |OCaml|
+   loadpath (cf. :cmd:`Declare ML Module`). It is available for
+   convenience, for instance for use in an interactive session, and it
+   is not exported to compiled files. For separation of concerns with
+   respect to the relocability of files, it is recommended to use
+   :n:`-I @string`.
 
 .. cmd:: Print ML Path
 
-   This command displays the current |OCaml| loadpath. This
-   command makes sense only under the bytecode version of ``coqtop``, i.e.
-   using option ``-byte``
-   (cf. :cmd:`Declare ML Module`).
-
+   This command displays the current |OCaml| loadpath, as provided by
+   the command :cmd:`Add ML Path @string` and by the command line
```

Comment:
I think it's a bad idea to explicitly mention `Add ML Path` here. Your change in documentation above explicitly states that it's equivalent to `-I` and it's not recommended to use it.

Reply:
Maybe at least change the order of `-I` vs `Add ML Path`.

Reply:
What do I do? I will a priori follow @Zimmi48's compromise? @ppedrot: Please tell if you want to insist.